www/layouts/educator/section.html

```diff
@@ -302,9 +306,9 @@ <h3>Explore Open Reflective Practice at MIT</h3>
                 <b>How would you like to explore open reflective practice at MIT?</b>
```

Comment:
@pdpinch should we also change this line ("How would you like to explore open reflective practice") to "How would you like to explore how we teach at MIT?"

Reply:
good catch